autofit/mapper/prior_model/array.py

```diff
@@ -0,0 +1,215 @@
+from typing import Tuple, Dict, Optional, Union
+
+from autoconf.dictable import from_dict
+from .abstract import AbstractPriorModel
+from autofit.mapper.prior.abstract import Prior
+import numpy as np
+
+from autofit.jax_wrapper import register_pytree_node_class
+
+
+@register_pytree_node_class
+class Array(AbstractPriorModel):
+    def __init__(
+        self,
+        shape: Tuple[int, ...],
+        prior: Optional[Prior] = None,
+    ):
+        """
+        An array of priors.
+
+        Parameters
+        ----------
+        shape : (int, int)
+            The shape of the array.
+        prior : Prior
+            The prior of every entry in the array.
+        """
+        super().__init__()
+        self.shape = shape
+        self.indices = list(np.ndindex(*shape))
```

Comment:
you are converting `self.indices` to a list but you are usint the typing `Tuple[int, ...]` everywhere.